### ISSUE TITLE
[ic-input-container] Investigate loki test failures when using ic-input-container component

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#256]
## Summary
When adding a new component in that uses ic-input-container, other components that also use it start failing visual tests even though they haven't been touched. 

## 💬 Description
Create a new component & have it render a ic-input-container. run the visual tests & select & textfield tests start failing, even though you have not touched those.

Spent some time looking into it and the ic-input-container actually does very little - the disabled & readonly props don't appear to have any effect, other than to add a class name name, but nothing is styled on that as far as i can tell.

So basically the component is just rendering a div with the following css

{ display: flex; flex-direction: column; position: relative; }
Potentially we could just remove that component and replace with divs in textfield & select which currently use it.

## 💰 Use value
Developers not having to worry about breaking other visual tests by using the ic-input-container.

**Comment by MI6-255** [comment]
Loki no longer relevant
